test/sql/chunk_utils.sql

```diff
@@ -171,8 +165,8 @@ SELECT * FROM show_chunks('drop_chunk_test2');
 
 \dt "_timescaledb_internal"._hyper*
 -- show_chunks and drop_chunks output should be the same
-\set QUERY1 'SELECT show_chunks(\'drop_chunk_test1\', older_than => 3)::NAME'
-\set QUERY2 'SELECT drop_chunks(3, \'drop_chunk_test1\')::NAME'
+\set QUERY1 'SELECT show_chunks(\'drop_chunk_test1\', older_than => 3)::REGCLASS::TEXT'
```

Comment:
More of this casting that seems unnecessary. I guess this applies also to a lot of places below.